core/primitives/src/block_header.rs

```diff
@@ -1044,13 +1044,13 @@ impl BlockHeader {
     }
 
     #[inline]
-    pub fn gas_price(&self) -> Balance {
+    pub fn next_gas_price(&self) -> Balance {
         match self {
             BlockHeader::BlockHeaderV1(header) => header.inner_rest.gas_price,
```

Comment:
Let's keep it consistent and rename it for all BlockHeader versions. That is also what I've done when adding `prev_` prefix: https://github.com/near/nearcore/pull/9500